app/uk/gov/hmrc/eoricommoncomponent/frontend/config/AppConfig.scala

```diff
@@ -59,6 +60,10 @@ class AppConfig @Inject() (
     case Journey.Subscribe => s"$feedbackLinkSubscribe-${service.code}"
   }
 
+  private lazy val displayEuLocationUntil: DateTime = DateTime.parse(config.get[String]("displayEuLocationUntil"))
```

Comment:
There is no point of doing this val lazy. Better to stay with just a val

Reply:
I changed it to lazy after an AppConfigSpec test failed.  Otherwise I'd have to mock a value for the date each time.  I presumed that's why the other vals are lazy.

Reply:
Yeah, lazy approach has been done for those values :D 

I think that with intialisation of config values is better to have them not lazy, I'm always worry about this, specially when the lazy val has some date, but it follows the approach that we have in our application so it's okay.

Thanks for explanation, I wasn't aware about it.

Reply:
Re:  "worry" - true.    With lazy initialisation it won't blow up in case of bad configuration until someone tries to access the user location page.

Might be worth a quick BAU PR to remove the laziness and fix the test properly.

Reply:
Yes, that's a good idea.